src/collectors/ebpf.plugin/ebpf.h

```diff
@@ -338,7 +338,7 @@ void ebpf_pid_file(char *filename, size_t length);
 #define EBPF_COMMON_DIMENSION_PACKETS "packets"
 #define EBPF_COMMON_DIMENSION_FILES "files"
 #define EBPF_COMMON_DIMENSION_MILLISECONDS "milliseconds"
-#define EBPF_COMMON_DIMENSION_KILLS "kills"
+#define EBPF_COMMON_DIMENSION_KILLS "kills/s"
```

Comment:
This is no longer "common" or "dimension", right? You use this define as chart unit only. I think it will be good to rename it. Likely true for other "EBPF_COMMON_DIMENSION_*" defines (but I didn't check).

Reply:
I agree with your point, and I already renamed for oomkill, because this PR is working exclusively with this thread. Next PR, that will work with remaining threads, I will work with others. :handshake: 

Best regards!